// Starter Code for Exercise 1
// From "Converting Exceptions with Try" atom
import com.atomicscala.AtomicTest._

f(0) is "OK Bob"
f(1) is "Reason"
f(2) is "11"
f(3) is "1.618"
